keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1109,21 +1357,16 @@ Below are some examples to consider, in addition to the aforementioned [maturity
 
 - Address reviews and bug reports from Alpha users
 - E2e tests are in Testgrid and linked in KEP
-- A scalability test to demonstrate the limited impact of the additional API call
-  when terminating a Pod
-- Re-evaluate modification to kubelet to send a dedicated condition when
-  terminating a Pod, based on user feedback (see: [Termination initiated by Kubelet](#termination-initiated-by-kubelet))
-- Re-evaluate supporting of `onExitCodes` when `restartPolicy=OnFailure` (see: [Relationship with Pod.spec.restartPolicy](#relationship-with-podspecrestartpolicy))
-- Re-evaluate introduction of a generic opinionated condition type
-  indicating that a pod should be retried (see: [Evolving condition types](#evolving-condition-types))
+- implementation of adding a dedicated pod condition by Kubelet when
+  terminating a Pod (see: [Termination initiated by Kubelet](#termination-initiated-by-kubelet))
 - Simplify the code in job controller responsible for detection of failed pods
```

Comment:
I think we can move this one to GA.

Reply:
I rephrased the point to make it clear what is behind (adding the DisruptionTarget or ResourceExhausted conditions 
depending on the scenario). The point was written first when we assumed there will be only one condition added by
kubelet so it could be misleading in its previous form.

As for moving it to GA, I think it would be good that users can experiment with the kubelet part of the feature
(especially graceful node shutdown and resource limits) in Beta.

Reply:
I meant "simplify the code in job controller" :)

kubelet should be in beta.

Reply:
Moved to GA.
I'm also wondering about moving "E2e tests are in Testgrid and linked in KEP" to GA, cause it requires "The feature flag enabled by default" to me merged-in first. If we wait with enabling the flag till other things are done it may be a tight schedule.

Reply:
You can update the KEP after the enhancements freeze to add the links. I would leave it as a beta requirement.